PowerEditor/src/NppBigSwitch.cpp

```diff
@@ -194,6 +194,50 @@ LRESULT Notepad_plus::process(HWND hwnd, UINT message, WPARAM wParam, LPARAM lPa
 		{
 			NppDarkMode::handleSettingChange(hwnd, lParam);
 
+			const bool enableDarkMode = NppDarkMode::isExperimentalActive();
+			if (NppDarkMode::isWindowsModeEnabled() && (enableDarkMode != NppDarkMode::isEnabled()))

```

Comment:
@ozone10 
Could you explain the reason of `(enableDarkMode != NppDarkMode::isEnabled())` ?

Reply:
@donho 
e.g. user has OS set to light mode, with Windows Mode enabled Notepad++ will start in light mode. Now user will manually enable dark mode in Notepad++. After specific time OS will switch to dark mode. But Notepad++ has already dark mode enabled, so do nothing.

Reply:
That should be a comment in the code I believe, or else someone in the future will probably remove it

Reply:
@donho 
add comment to be sure
```
// Windows mode is enabled
// and don't change if Notepad++ is already in same mode as OS after changing OS mode
```